docs/source/cookbook/dataframes.rst

```diff
@@ -0,0 +1,166 @@
+Using external data
+-------------------
+
+It's not always practical or desirable to store all the data required by a model in the JSON document. This is particularly true when working with large amounts of timeseries data, but is also applicable to data better stored in a tabular format.
+
+Many parameter types support loading data from an external file using the ``"url"`` keyword. The data is loaded during the setup/reset process, before the model is run.
+
+Supported formats
+=================
+
+Support for the following data formats is available via the `Pandas <http://pandas.pydata.org/pandas-docs/stable/io.html>`_ module.
+
+    * Comma separated values (.csv)
+    * Excel (.xls, .xlsx)
+    * HDF5 (.h5)
+
+**Warning:** When reading Excel documents formulae are supported but not re-evaluated and links are not updated; the value read is the value present when the document was last saved.
+
+When working with large amounts of timeseries data the HDF5 is recommended as it has superior read speeds. Data in an HDF5 file should be in the "fixed" format; this is achieved by passing the ``format="fixed"`` argument to ``DataFrame.to_hdf``.
```

Comment:
Perhaps we should reference `TablesArrayParameter` for using HDF5 through `pytables` (rather than pandas) more directly.